oaknational/Oak-Web-Application
# [CUR-301] Added Avo tracking to curric pages

## Description

- Added the following tracking events:

### Unit Tab
- Curriculum Thread Highlighted
- Year Group Selected 

### Unit Modal
- Unit modal is opened

### Subject Phase Picker
- Curriculum Visualiser is accessed through subject phase picker


## How to test

1. Go to https://deploy-preview-2039--oak-web-application.netlify.thenational.academy
2. Click on the curriculum visualiser
3. You should see no change in behaviour to any of the above events
4. If using the console, you should be able to see events sent through like below 

<img width="1433" alt="Screenshot 2023-10-24 at 11 17 23" src="https://github.com/oaknational/Oak-Web-Application/assets/32605982/6cbbda75-2b25-4cf2-a80f-2f8880e7b6da">


## Checklist
